<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paciente_ui</class>
 <widget class="QWidget" name="paciente_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="3" column="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Paciente Actual :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_fichaCurrentPatient">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Historia Clínica : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="lineEdit_CurrentPatient">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>L</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ñ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>R</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>T</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>U</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>W</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Z</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Orden Alfabético:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QLineEdit" name="lineEdit_buscar">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string extracomment="Apellidos y Nombres o Historia Clinica"/>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BUSCAR:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <spacer name="horizontalSpacer_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="0" column="0" rowspan="5">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="Line" name="line_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="Line" name="line_4">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QPushButton" name="verDatos_paciente">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&amp;Ver Datos</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/preferences-contact-list.png</normaloff>:/new/preferences-contact-list.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_editarDatos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>E&amp;ditar Datos</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/sign-up.png</normaloff>:/new/sign-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="verConsultas_paciente">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Ver &amp;Consultas</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/distributor-report.png</normaloff>:/new/distributor-report.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="eliminar_paciente">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elimina&amp;r Paciente</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/Remove-Male-User.png</normaloff>:/new/Remove-Male-User.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>228</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="newConsulta_paciente">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&amp;Nueva Consulta</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/sign-up.png</normaloff>:/new/sign-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="newPaciente_paciente">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nuev&amp;o Paciente</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/Add-Male-User.png</normaloff>:/new/Add-Male-User.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="6" colspan="2">
    <widget class="Line" name="line_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QTableView" name="tableView_ListaPacientes1">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
